lib/spack/spack/modules/common.py

```diff
@@ -470,6 +471,11 @@ def hash(self):
             return self.spec.dag_hash(length=hash_length)
         return None
 
+    @property
+    def conflicts(self):
+        """Conflicts for this module file"""
+        return self.conf.get("conflict", [])
```

Comment:
This is fine, but now we can also remove the same property in `TclConfiguration`.

Reply:
done